### DISCUSSION TITLE
Welcome to dVoting Discussions!

### DISCUSSION THREAD

**Discussion by arlbibek** [arlbibek/dVoting#2] [General]
## 👋 Welcome to dVoting!
  Setting this Discussion up as a place to have a conversation concerning the project with other members. We hope that you:
  * Ask questions you’re wondering about the project.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below with an introduction of yourself and tell us about what you do and your thoughts regarding the project. ✍

**Comment by Chamallaw** [comment]


**Reply by arlbibek** [reply]
By default, the one who deploys the smart contract to the network is the administrator. Hence, **you cannot specify the administrator**.
